### DISCUSSION TITLE
Every second row is empty

### DISCUSSION THREAD

**Discussion by kadartamascsaba** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#418] [Q&A] [answered]
What is the root cause if I try to fill up a LED panel row by row, and every second row will be empty?

**Comment by kadartamascsaba** [comment]
I have tried to use this single 32x16 LED matrix, with a ESP32 wroom-32. I used the default ESP32 pins, it connected correctly, but I only saw every second row. Is it a LED matrix or ESP32 problem?

**Reply by board707** [reply]
> Is it a LED matrix 

What code is written on the back of the matrix?
Something like P10-3216-2727-S4-V1
Or show the photo

**Reply by kadartamascsaba** [reply]
![image](https://user-images.githubusercontent.com/15170619/225845061-57faab6a-aef5-4700-a08f-9fcce97ceca9.png)

I just simple tried to draw all of the pixels like row by row and it looks like draw the first row after it again (I mean, the leds looks like brighter) and after it the third row, etc.

**Comment by board707** [comment, marked as answer]
>  it looks like draw the first row after it again (I mean, the leds looks like brighter) and after it the third row, etc.

Check the connection of the A line
